### PR TITLE
Update to community 2.15

### PR DESCRIPTION
**Pull request by ahousing**
Note that I couldn't compile netx locally (the last stage failed).  But, this is true without any changes.  The purpose of these changes is to prepare to consume 2.16.

### CODE REVIEW

File: http.bridge/itest/pom.xml [hunk 1]
```diff
@@ -83,23 +82,23 @@
...
-                            <artifactId>netx.http.bridge</artifactId>
-                            <version>${project.version}</version>
-                            <type>jar</type>
-                            <destFileName>netx.http.bridge.jar</destFileName>
-                        </artifactItem>
-                    </artifactItems>
-                </configuration>
                 <executions>
                     <execution>
+                        <phase>package</phase>
```
Comment (dpwspoon) [inline http.bridge/itest/pom.xml]: Why did you add the phase?  (Probably fine, just wasn't sure why)

Comment (ahousing) [inline http.bridge/itest/pom.xml]: Maven weirdness.  Since I define a phase for the maven-dependency-plugin for the license, I have a feeling that that phase was (for some unknown reason) being used for this execution as well.  I did a google search for the default phase for copy and added it.

Comment (dpwspoon) [inline http.bridge/itest/pom.xml]: Looks like [default phase](http://maven.apache.org/plugins/maven-dependency-plugin/copy-mojo.html) is process-sources, which I assume is what it was on before.  I also don't think it should be necessary to do this as maven is convention over configuration so default configuration should (most likely) be correct, unless we are doing something peculiar (read, potentially non standard and wrong)

**Comment by ahousing** [comment]
I couldn't reproduce the problem that I was seeing yesterday.  Removed the phase.  If it comes back, we'll re-discuss.
